swarm/network/stream/snapshot_retrieval_test.go

```diff
@@ -284,29 +284,35 @@ func runRetrievalTest(t *testing.T, chunkCount int, nodeCount int) error {
 			return err
 		}
 
+		for _, v := range conf.hashes {
+			log.Debug("conf.hashes", "ref", v)
+		}
+
 		// File retrieval check is repeated until all uploaded files are retrieved from all nodes
 		// or until the timeout is reached.
 	REPEAT:
```

Comment:
maybe we should get rid of these labels?

Reply:
On a second look, the modification doesn't really affect the labels. The idea here is that if one of the chunks errors, you want to start the check from scratch. So we can't really remove REPEAT, or this will change the logic of the test.